.changeset/tidy-suits-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-foundations': minor
```

Comment:
if we think this could break layouts, it should be a major (see https://github.com/guardian/source/blob/main/docs/contributing/overview.stories.mdx#versioning)

```suggestion
'@guardian/source-foundations': major
```